lib/models/persistent_tab_controller.dart

```diff
@@ -2,10 +2,17 @@ part of "../persistent_bottom_nav_bar_v2.dart";
 
 /// Navigation bar controller for `PersistentTabView`.
 class PersistentTabController extends ChangeNotifier {
-  PersistentTabController({int initialIndex = 0})
-      : _index = initialIndex,
-        assert(initialIndex >= 0, "Nav Bar item index cannot be less than 0");
+  PersistentTabController({
+    int initialIndex = 0,
+    int historyLength = 5,
+  })  : _initialIndex = initialIndex,
+        _historyLength = historyLength,
+        _index = initialIndex,
+        assert(initialIndex >= 0, "Nav Bar item index cannot be less than 0"),
+        assert(historyLength >= 0, "Nav Bar history length cannot be less than 0");
 
+  final int _initialIndex;
+  final int _historyLength;
```

Comment:
It would be nice to have a small doc comment with the behavior you described in https://github.com/jb3rndt/PersistentBottomNavBarV2/pull/138#issuecomment-2043481131